Dart/src/com/jetbrains/lang/dart/ide/generation/CreateConstructorFix.java

```diff
@@ -44,10 +44,9 @@ protected Template buildFunctionsText(TemplateManager templateManager, Set<DartC
         template.addTextSegment(",");
       }
     }
-    template.addTextSegment(")");
-    template.addTextSegment("{\n");
+    template.addTextSegment(");");
```

Comment:
I guess you are right, just wanted to make sure, is constructor without body more popular use case?